runtime/vm/jnicsup.cpp

```diff
@@ -2545,7 +2545,6 @@ defineClass(JNIEnv *env, const char *name, jobject loader, const jbyte *buf, jsi
 		J9TranslationBufferSet *dynamicLoadBuffers;
 		UDATA classNameLength;
 		U_8 * className;
-		U_8 check;
 		U_8 c;
 		U_8 * checkPtr;
```

Comment:
`check` must stay because it should be used on 2578 which should have been
```
		if (0 != (check & 0x80)) {
```
While you're here, please add initialization for these local variables:
```
		J9ClassLoader * classLoader = NULL;
		J9TranslationBufferSet *dynamicLoadBuffers = NULL;
		UDATA classNameLength = 0;
		U_8 * className = NULL;
		U_8 check = 0;
		U_8 c = 0;
		U_8 * checkPtr = NULL;
```